src/test/java/uk/gov/hmcts/reform/civil/service/robotics/mapper/EventHistoryMapperTest.java

```diff
@@ -2522,9 +2636,9 @@ void shouldPrepareExpectedEvents_whenCaseTakenOfflineAfterClaimDetailsNotifiedEx
                                   .agreedExtensionDate(caseData.getRespondentSolicitor1AgreedDeadlineExtension()
                                                            .format(ISO_DATE))
                                   .build())
-                .eventDetailsText(format("agreed extension date: %s", caseData
+                .eventDetailsText(format("agreedExtensionDate: %s", caseData
```

Comment:
these changes need to be reverted, the RPA string message will read:
"agreedExtensionDate: 2022 03 29" which is incorrect. The business wanted this updated a few sprints ago to read in the UK data format!